### ISSUE TITLE
Export Error with multiple art in F.SilkS

### ISSUE THREAD

**Issue by iharshadec** [badgeek/svg2shenzhen#88]
svg2shenzhen - 0.2.18.5
inkscape - 1.0
OS - Windows 10

Export Settings
![image](https://user-images.githubusercontent.com/20681096/90660165-30684180-e263-11ea-807c-820648222d05.png)

Export Error
![image](https://user-images.githubusercontent.com/20681096/90660137-26ded980-e263-11ea-9128-dcaeee713956.png)

.svg file
[pcbArt.zip](https://github.com/badgeek/svg2shenzhen/files/5097665/pcbArt.zip)

**Comment by badgeek** [comment]
@iharshadec i just checked your file and the layers is not properly named, i need file with proper layer

**Comment by iharshadec** [comment]
I have generated layers with prepare document options only.
There is all layers that I did checkboxes while preparing document. In addition to that I have added Extra layer to put some extra/tirals on it.

Layers
![image](https://user-images.githubusercontent.com/20681096/90714573-9be2fb00-e2c5-11ea-862d-a2699c25fdd3.png)

Objects
![image](https://user-images.githubusercontent.com/20681096/90714584-a43b3600-e2c5-11ea-8ac0-0de890735983.png)
